### ISSUE TITLE
MySensors incorrect roller shutters children IDs

### ISSUE THREAD

**Issue by bywciu** [domoticz/domoticz#4295]
Hi, I've found rather strange issue and I can't find a solution.
I have multinode which handles my roller shutters.
In "Hardware" section, everything is fine, all children have correct IDs (the ones which I set in the sketch), but in "Devices" tab, "Unit" values are wrongly assigned. When child ID in my sketch is set for more than 15, it starts counting devices from 0 till 15 (in random order). Also when I add more than 16 roller shutters (so IDs from 0-15 are used), they are just not there. No device created.

So if I have:
`present(1, S_COVER, "Shutter 1");`
`present(2, S_COVER, "Shutter 2");`
everything is fine, children have IDs 1 and 2, but when I do this:
`present(20, S_COVER, "Shutter 1");`
`present(21, S_COVER, "Shutter 2");`
 children have IDs 0 and 1 (in random order, so 20 can be 1 and 21 can be 0)

It seems to be blinds component issue rather than MySensors implementation itself, but I don't have any other hardware to check.

**Comment by gizmocuz** [comment]
@bywciu , Internally in MySensorsBase.cpp the ChildID seems to be a integer and should be able to be much larger then 15
Is it possible to build Domoticz yourself (Visual Studio 2019) and set some breakpoints to debug this ?

**Comment by quexpl** [comment]
Hi, I have the same problem.

I set up 22 of `S_COVER` and the rest are `S_BINARY` for the test. In the Hardware page everything seems to be fine:
![image](https://user-images.githubusercontent.com/8268830/97166081-f34d8d80-1784-11eb-9d61-d25cf20b5931.png)

but there is something wrong with the devices page: 
![image](https://user-images.githubusercontent.com/8268830/97166203-2b54d080-1785-11eb-985f-2c20c22f11c8.png)

I complied the most recent version from the development branch, and set some extra debug logs in the LoadDevicesFromDatabase(). The ChildID is correct, and it returns all devices. https://github.com/domoticz/domoticz/blob/316c9b7c7143f68c9effa5cac80d35a0e75c6816/hardware/MySensorsBase.cpp#L253
What else can I check? How to debug this? Where should I look?
